source/upgrade/important-upgrade-notes.rst

```diff
@@ -24,7 +24,11 @@ Important Upgrade Notes
 |                                                    | Migration times: On a system with 12M posts, and 1M fileinfo entries, the migration takes 15s. This migration is non-locking. Note that there is no migration    |
 |                                                    | for MySQL deployments because this optimization is only applicable for PostgreSQL.                                                                               |
 +----------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| v10.5                                              | The internal workings of the `PluginLinkComponent` in the web app have been changed to unmount link tooltips from the DOM by default, significantly improving    |
+| v10.5                                              | Both Mattermost v10.5.0 and v10.5.1 come bundled with a Jira plugin version that contains a bug causing the plugin configuration to disappear. We strongly       |
+|                                                    | discourage upgrading to these versions to avoid this issue. If you have already upgraded to 10.5.0 or 10.5.1, we recommend downgrading the Jira plugin to a      |
```

Comment:
same as in the other upgrade notes. instead of recommending downgrading we should encourage downgrading to jira 4.2.1 or mattermost 10.5.2